File: GFG/Python/DSA/Easy/LCM_GCD.py
def lcmAndGcd(A, B):
    # code here
    a = A
    b = B

    while a > 0 and b > 0:
        if a > b:
            a = a % b
        else:
            b = b % a

    gcd = a if a > 0 else b

    lcm = abs(A*B)/gcd

    return int(lcm), gcd

File: GFG/JavaScript/DSA/Easy/LCM-GCD.js
function lcmAndGcd(a, b) {
  //code here
  A = a;
  B = b;
  while (a > 0 && b > 0) {
    if (a > b) a = a % b;
    else b = b % a;
  }
  let gcd = a > 0 ? a : b;

  let lcm = (A * B) / gcd;
  return [lcm, gcd];
}

console.log(lcmAndGcd(6, 10));
